README.md

```diff
@@ -2319,6 +2319,41 @@ ANSWERS.addComponent('MyCustomComponent', {
 });
 ```
 
+# Custom Partials
+
+[Handlebars partials](https://handlebarsjs.com/guide/partials.html) are
+templates that can be reused and called directly from other templates. The 
+SDK offers an `ANSWERS.registerPartial` function. This will map a partial string 
```

Comment:
nit: Since this repo is no longer officially called the SDK, I would reword this to "The ANSWERS object offers an `ANSWERS.registerPartial` function for custom partials."